app/views/corporate/extra/_tool.erb

```diff
@@ -1,3 +1,9 @@
+<h2 id="get-this-tool" role="heading" aria-level="2">
```

Comment:
There's a `heading_tag` helper we've got that could generate these for you. Example here: https://github.com/moneyadviceservice/frontend/blob/master/app/views/campaigns/_section.html.erb#L4